R/udf_array_details.R

```diff
@@ -54,13 +54,11 @@ UDFArrayDetails <- R6::R6Class(
       }
       if (!is.null(self$`ranges`)) {
         UDFArrayDetailsObject[['ranges']] <-
-          ####self$`ranges`$toJSON()
-          jsonlite::toJSON(as.integer(serialize(self$ranges, NULL)))
+          self$`ranges`$toJSON()
```

Comment:
This does not need backtick as `ranges` is a normal identifier string.

Reply:
thanks @eddelbuettel ! I'll edit this & some in the vicinity but most of these are among the ~30KLOC generated by OpenAPI which backticks everything in an overabundance of caution. :(

Reply:
Oh, I missed that.  If it is robot-generated (which may unconditionally toss backticks in) then never mind.  Not worth chasing.

Reply:
Euh, it was me as a non-human hand-editing what a robot had written. :D